R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
I didn't see that as a requirement from the manual. Do we only need a single \name field for an Rd? I guess the duplicated alias issue is caused by multiple documents pointed by the same method if we use different @rdname and @name?